src/layouts/settings/SiteSettings.tsx

```diff
@@ -28,12 +32,12 @@ export const SiteSettings = () => {
       <Switch
         label={t("sitesettings.header")}
         onChange={() => dispatch(setSiteSettings({ allow_registration: !siteSettings.allow_registration }))}
-        checked={siteSettings.allow_registration}
+        checked={allowRegistration}
```

Comment:
why do we no longer listen to the site settings here?

Reply:
I think the issue was, that it didn't update in time. It resulted in values going back to the old value and then to the new value when saving. Now it's in a state and does update correctly.

Reply:
That's correct :-)